pkg/models/manager.go

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+type Manager interface {
```

Comment:
Rename the interface or methods in v2 Manager model.
Manager is a general term.  If it is only for managing AutoEvent, it should be called AutoEventManager.
Or, we should rename the RestartForDevice to RestartAutoEventForDevice